### ISSUE TITLE
Exception thrown at launch in version 1.4.27

### ISSUE THREAD

**Issue by mesontau** [HUPO-PSI/mzIdentML#95]
I got an exception thrown when launching v 1.4.27 at a Windows 7 machine (Java version 1.8.0_144). See below the exception report:

```
Exception in thread "main" java.lang.NoClassDefFoundError: org/slf4j/MarkerFactory
        at org.apache.log4j.Category.<clinit>(Category.java:53)
        at psidev.psi.pi.validator.MzIdentMLValidatorGUI.<clinit>(MzIdentMLValidatorGUI.java:66)
Caused by: java.lang.ClassNotFoundException: org.slf4j.MarkerFactory
        at java.net.URLClassLoader.findClass(Unknown Source)
        at java.lang.ClassLoader.loadClass(Unknown Source)
        at sun.misc.Launcher$AppClassLoader.loadClass(Unknown Source)
        at java.lang.ClassLoader.loadClass(Unknown Source)
        ... 2 more
```

I just downloaded the v1.4.27 snapshot and executed the bat file right away, so I am probably missing something very obvious. Any idea?

Thanks! 
Pedro J.

**Comment by smdb21** [comment]
I got the same thing

**Comment by UKQIDA** [comment]
Same here!

**Comment by ciesinsk** [comment]
Same here.

**Comment by mesontau** [comment]
It looks then a general problem, and not a particular configuration issue at my computer. @germa @andrewrobertjones is there anything we are missing here?

**Comment by germa** [comment]
Corrected in v1.4.28
